### PR TITLE
Adding volunteer, ready-to-startinfo to organizer edit mission screen

### PR DESCRIPTION
**Pull request by laredotornado**
Based on this GitHub issue

https://github.com/factn/resilience-app/issues/561

### CODE REVIEW

File: src/app/dashboard/Missions/MissionEditView.jsx [hunk 1]
```diff
@@ -211,9 +217,17 @@ const MissionEditView = ({ mission, toDetailsView, toListView }) => {
...
+    volunteers.find((el) => el.id === mission.volunteerUid)
+  );
 
   const props = { classes, mission };
 
+  const volunteerEditable =
+    mission.status === MissionStatus.unassigned ||
+    mission.status === MissionStatus.tentative ||
+    mission.status === MissionStatus.assigned;
+
```
Comment (mat10tng) [inline src/app/dashboard/Missions/MissionEditView.jsx:230]: outside of the class
`const volunteerStatus = [MissionStatus.unassigned, MissionStatus.tentative, MissionStatus.assigned]`
inside of the class
`const volunteerEditable = volunteerStatus.includes(mission.status)`

File: src/app/dashboard/Missions/MissionEditView.jsx [hunk 2]
```diff
@@ -246,6 +264,31 @@ const MissionEditView = ({ mission, toDetailsView, toListView }) => {
...
+          : "",
+      tentativeVolunteerDisplayName:
+        selectedVolunteer && values.status === MissionStatus.tentative
+          ? selectedVolunteer.volunteerDisplayName
+          : "",
+      tentativeVolunteerPhoneNumber:
+        selectedVolunteer && values.status === MissionStatus.tentative
+          ? selectedVolunteer.volunteerPhoneNumber
+          : "",
+      readyToStart: values.readyToStart,
```
Comment (mat10tng) [inline src/app/dashboard/Missions/MissionEditView.jsx:291]:
```
const getVolunteerAttribute = (selectedVolunteer, status, attr) => {
    if(selectedVolunteer && status === MissionStatus.tentative){
        return selectedVolunteer[attr]
   }
   return ""
}

{...,
tentativeVolunteerUid = getVolunteerAttribute(selectedVolunteer, values.status, "volunteerUid")
}
```

File: src/app/dashboard/Missions/MissionEditView.jsx (hunk 1 above)
Comment (laredotornado) [reply src/app/dashboard/Missions/MissionEditView.jsx:236]: Fixed!

File: src/app/dashboard/Missions/MissionEditView.jsx (hunk 2 above)
Comment (laredotornado) [reply src/app/dashboard/Missions/MissionEditView.jsx:309]: Had to add one more param to the "getVolunteerAttribute" function but should capture what you are pointing out.

Comment (mat10tng) [reply src/app/dashboard/Missions/MissionEditView.jsx:309]: It did not look as pretty as I had hope, my bad there, lets keep it for now

**Pull request merged**